### PR TITLE
Feature/product component

### LINKED ISSUE #6: Storybook: Product component
## User story

**Who:** **As a** developer
**What:** **I want to** be have a Product component where can have Image of the product, Name, Price and have a button to add it to the shopping card and finally a favourite heart to add the product to favourites
**Why:** **so that I can** have the same component in multiple pages for multiple products.

## Acceptance criteria

Create a storybook component like the one we can see below:
 
Screen Shot 2022-01-20 at 10 54 06

Check the wireframes here: https://www.figma.com/file/yo4QT2EcPo83HAMI3ydMxb/HYF-Wireframes?node-id=4%3A1324

## Implementation details

Use **Knobs** where we can choose  different **Images**, **Names** and **Prices**. For more info check [Figma](https://www.figma.com/file/yo4QT2EcPo83HAMI3ydMxb/HYF-Wireframes?node-id=4%3A1324)

Download the images from the wireframes and add them to this folder src/client/assets/images

### PR DESCRIPTION
**Pull request by hparul**
# Description

This PR is about creating a Product component which is reusable.

Fixes # (issue)
 https://github.com/HackYourFuture-CPH/fp-class19/issues/6

# How to test?

Please provide a short summary how your changes can be tested?
1. run -npm run story
2. go to product component and verify that a product component is displayed with default values.

# Checklist

- [x] I have performed a self-review of my own code
- [x] I have followed the name conventions for CSS Classnames and filenames, Components names and filenames, Style filenames, if you are in doubt check the the project README.MD and here https://github.com/HackYourFuture-CPH/curriculum/blob/master/review/review-checklist.md
- [x] I have commented my code, particularly in hard-to-understand areas, if you code was simple enough mark the box anyway
- [x] I have made corresponding changes to the documentation, if you code was simple enough mark the box anyway
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Review by dpfernandes — CHANGES_REQUESTED** [review]
Please create a PR just for the images. Don't use spaces when naming the files.

**Review by Divyajg — COMMENTED** [review]
Looks good in my machine. Try to solve the conflicts. it might good to go.

**Comment by suvarna-ratna** [comment]
super and delete branch as well  but have to wait atleast two approvals before merging ...

**Pull request merged**
